nilearn/plotting/displays/_figures.py

```diff
@@ -103,3 +113,141 @@ def savefig(self, output_file=None):
         self._check_output_file(output_file=output_file)
         if self.figure is not None:
             self.figure.write_image(self.output_file)
+
+    def add_contours(self, roi_map, levels=None, labels=None, lines=None):
+        """
+        Draw boundaries around roi.
+
+        Parameters
+        ----------
+        roi_map : str or :class:`numpy.ndarray` or list of
+            :class:`numpy.ndarray` ROI map to be displayed on the surface
+            mesh, can be a file (valid formats are .gii, .mgz, .nii,
+            .nii.gz, or Freesurfer specific files such as .annot or .label),
+            or a Numpy array with a value for each vertex of the surf_mesh.
+            The value at each vertex one inside the ROI and zero inside ROI,
+            or an integer giving the label number for atlases.
```

Comment:
parameter type definition and description need to go on separate lines. Also a multi-line type definition needs a `\` at the end of the line. See https://github.com/nilearn/nilearn/blob/1759fa2f719e8eb56627698484f8665cc5336824/nilearn/glm/first_level/first_level.py#L504-L508